tests/foreman/maintain/test_upgrade.py

```diff
@@ -127,11 +127,16 @@ def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
     last_y_stream = last_y_stream_version(settings.server.version.release)
     custom_host.download_repofile(product='satellite', release=last_y_stream)
     custom_host.execute('dnf -y module enable satellite:el8 && dnf -y install satellite')
-    # Install without system checks to get around installer checks
+    # Install with development tuning profile to get around installer checks
     custom_host.execute(
-        f'satellite-installer --scenario satellite --disable-system-checks --tuning {profile}',
+        'satellite-installer --scenario satellite --tuning development',
         timeout='30m',
     )
+    # Change to correct tuning profile (default or medium)
+    custom_host.execute(
+        f'sed -i "s/tuning: development/tuning: {profile}/g" '
+        '/etc/foreman-installer/scenarios.d/satellite.yaml'
```

Comment:
```suggestion
        f'sed -i "s/tuning: development/tuning: {profile}/g" {SATELLITE_ANSWER_FILE}'
```

Reply:
oops, this is not correct. Nevertheless, could you add the path to this file to constants too, please?